thumbor/transformer.py

```diff
@@ -343,8 +343,10 @@ def fit_in_resize(self):
         # ensure that filter should work on the real image size and not on the request
         # size which might be smaller than the resized image in case `full-fit-in` is
         # being used
-        self.context.request.width = int(max(self.context.request.width, resize_width))
-        self.context.request.height = int(max(self.context.request.height, resize_height))
+        requested_width = source_width if self.context.request.width == 'orig' else self.context.request.width
```

Comment:
fixes https://github.com/thumbor/thumbor/pull/877#discussion_r144110881

Reply:
good catch... completely forgot about that.. shall we maybe add a test next to https://github.com/thumbor/thumbor/pull/877/commits/28d9eab34d7bbd630897d4dae85f11058196de5e to check that this behaviour is correct?

Reply:
done